geomstats/geometry/connection.py

```diff
@@ -125,6 +125,22 @@ def exp(
         exp : array-like, shape=[..., dim]
             Point on the manifold.
         """
+        base_point = gs.to_ndarray(base_point, to_ndim=2)
```

Comment:
Following a similar [implementation](https://github.com/geomstats/geomstats/pull/1514#discussion_r866068967) by @ymontmarin, I think all this new code can be replaced by:

```base_point = gs.broadcast_to(base_point, tangent_vec.shape)```

I think it accomplishes the goals we're looking for.

Reply:
Yes, I remember. I think this solution probably works there also.